command/agent/agent_auto_auth_self_heal_test.go

```diff
@@ -388,20 +357,28 @@ func Test_NoAutoAuthSelfHealing_BadPolicy(t *testing.T) {
 	require.Contains(t, tokenSecret.Auth.Policies, policyName)
 	newToken := tokenSecret.Auth.ClientToken
 
-	// Write token to file
-	err = os.WriteFile(pathVaultToken, []byte(token), 0o600)
+	// Write new token to token file (where Agent would re-auto-auth from if
+	// it were triggered)
+	err = os.WriteFile(pathVaultToken, []byte(newToken), 0o600)
 	require.NoError(t, err)
 
 	// Wait for any potential *incorrect* re-triggers of auto auth
-	time.Sleep(secretRenderInterval * 3)
```

Comment:
I replaced this as it was a little misleading - auto_auth doesn't interact with the static secret render interval